### DISCUSSION TITLE
Import with tilde (~) path resolver in parcel2

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [parcel-bundler/parcel#5233] [Q&A]
In parcel 1, I was able to use tilde which usually refers to the nearest directory or the root directory of the project. But, I recently upgraded to parcel 2 and it's complaining. How do I enable the ~ path resolvers in a typescript/react repo ? Do I need to write a resolver for that or is there any build-in support ? Is there any example for this setup ?

**Comment by mischnic** [comment]
`~` refers to a slightly different folder in Parcel 2: https://v2.parceljs.org/features/module-resolution/#tilde-paths

**Reply by KaKi87** [reply]
Hello,

I don't know what that link which doesn't work was supposed to tell, but what I found at [here](https://parceljs.org/features/dependency-resolution/#tilde-specifiers) is :
> Tilde specifiers start with ~, and resolve relative to the nearest package root from the importing file. A package root is a directory with a package.json file, which would typically be found in node_modules, or as the root of a package in a monorepo. Tilde specifiers are useful for similar purposes as absolute specifiers, but are more useful when you have more than one package.

After reading that piece of documentation, I wrote the following code :
```html
<style>
	@import '~/node_modules/destyle.css/destyle.min.css';
</style>
```
In order to replace the following one :
```html
<style>
	@import '../../node_modules/destyle.css/destyle.min.css';
</style>
```
But, I'm getting the following errors :
```
@parcel/core: Failed to resolve '~/node_modules/destyle.css/destyle.min.css' from './components/App.vue'
```
```
@parcel/resolver-default: Cannot load file '../node_modules/destyle.css/destyle.min.css' in './components'.
```
My `parcel` version is `2.7.0`.

Thanks
